unified-runtime/include/ur_api.h

```diff
@@ -12320,6 +12323,9 @@ typedef union ur_exp_launch_property_value_t {
   /// [in] non-zero value indicates the amount of work group memory to
   /// allocate in bytes
   size_t workgroup_mem_size;
+  /// [in] non-zero value indicates a opportunistic native queue serialized
```

Comment:
```suggestion
  /// [in] non-zero value indicates an opportunistic native queue serialized
```

Reply:
Thanks I updated script to generate this change.